### ISSUE TITLE
Add a command `zetae2e run approvals` to set all approvals once

### ISSUE THREAD

**Issue by lumtis** [zeta-chain/node#2535]
**Describe the Issue**

Different E2E tests requires giving approvals to some specific contracts.

The `approve` function is currently run when necessary in each tests. This represent unnecessary functions to call as we could just set one big approval for all tests.
This is especially true for live testing on Ethereum when gas can be expensive.

**Idea**

- Introduce a separate command (as a E2E test itself) `approvals` that allow to set approval for a big amount for all used contracts in the E2E tests
- Remove usage of `approval` in all tests

**Comment by lumtis** [comment]
The new v2 E2E tests have a standardized way to use approval only once, no longer needed
